frontend/src/components/scan/ScanForm.jsx

```diff
@@ -213,15 +215,23 @@ export default function ScanForm() {
         if (!values.file) {
           errors.file = "required";
         }
-      } else if (values.observable_name) {
-        const pattern = RegExp(
-          observableType2PropsMap[values.classification].pattern
+      } else if (values.observable_names) {
+        const ObservableNamesErrors = values.observable_names.map(
+          (ObservableName) => {
+            const pattern = RegExp(
+              observableType2PropsMap[values.classification].pattern
+            );
+            if (!pattern.test(ObservableName)) {
+              return `invalid ${values.classification}`;
+            }
+            return null;
+          }
         );
-        if (!pattern.test(values.observable_name)) {
-          errors.observable_name = `invalid ${values.classification}`;
-        }
+
+        if (ObservableNamesErrors.some((e) => e))
+          errors.observable_names = ObservableNamesErrors;
```

Comment:
I think a comment helps to make this code easier to understand: you create an error list and then check with `some()` if at least an error exist, this is fine but at first sight it's not so clear

Reply:
Done. Does it look good?